packages/client/src/object/convertWireToOsdkObjects.ts

```diff
@@ -153,10 +200,20 @@ export async function convertWireToOsdkObjectsInPlace(
       });
     }
   }
+}
+
+/**
+ * @param objs the objects to be converted, the contents of this array will be mutated
+ */
+export async function convertWireToOsdkObjectsInPlace(
+  client: MinimalClient,
+  objs: OntologyObjectV2[],
+) {
+  fixObjectPropertiesInline(objs);
 
   // We dont want to refetch for each object type in this conversion
   const localObjectCache = createAsyncCache((client, apiName: string) =>
-    client.ontology.provider.getObjectOrInterfaceDefinition(apiName)
+    client.ontology.provider.getObjectDefinition(apiName)
```

Comment:
Shouldn't `localObjectCache` have a longer lifetime than just a single call to `convertWireToOsdkObjectsInPlace`? Or do we really really need to have the newest definitions?

Or am I misunderstanding this, and its actually a second layer of caching (if so, then why?).